js/index.js

```diff
@@ -26,17 +46,53 @@ function removeProduct(event) {
   const target = event.currentTarget;
   console.log('The target in remove is:', target);
   //... your code goes here
+  const toRemove = target.parentNode.parentNode.querySelector(".quantity input");
+  toRemove.value = 0
+  calculateAll();
 }
 
 // ITERATION 5
 
 function createProduct() {
-  //... your code goes here
+  const parent = document.getElementsByTagName("tbody")[0];
+  let productName = document.querySelector(".name1 input").value;
+  let priceNew = document.querySelector(".price1 input").value;
+  let row = parent.insertRow();
+  let nameCell = row.insertCell();
+  let priceCell = row.insertCell();
+  let quantityCell = row.insertCell();
+  let subtotalCell = row.insertCell();
+  let actionCell = row.insertCell();
+  row.className = "product"
+  nameCell.className = 'name';
+  priceCell.className = 'price';
+  quantityCell.className = 'quantity';
+  subtotalCell.className = 'subtotal';
+  actionCell.className = 'action';
+  nameCell.innerHTML = `<span> ${productName} </span>`;
+  priceCell.innerHTML = `$<span>${Math.round(priceNew*100)/100} </span>`;
+  quantityCell.innerHTML = `<input type = "number" value = "0" min = "0" placeholder = "Quantity" />`;
+  subtotalCell.innerHTML = ` $<span>0</span>`;
+  actionCell.innerHTML = `<button class = "btn btn-remove"> Remove </button>`;
+  let removeButton = actionCell.getElementsByTagName("button")[0];
+  removeButton.className = "btn btn-remove";
+  document.querySelector(".name1 input").value = "";
+  document.querySelector(".price1 input").value = 0;
+
 }
 
 window.addEventListener('load', () => {
   const calculatePricesBtn = document.getElementById('calculate');
   calculatePricesBtn.addEventListener('click', calculateAll);
-
-  //... your code goes here
-});
+  const removeBtn = document.getElementsByClassName("btn btn-remove");
+  const addRemoveButtons = () => {
```

Comment:
`
  for (const item of removeBtn){
    item.addEventListener(`click`, removeProduct);
  }
`
para que cuando cargue la pagina los botones de remover tenga la el listenner  y funcionen ya que 
no funcionara hasta que demos en crear un nuevo producto